doc/user_guide/make_win.sh

```diff
@@ -8,16 +8,19 @@ declare -a exts=(txt ps ds)
 # Functions definition ---------------------------------------------------------
 # Subroutine to remove files.
 clean_files () {
-	# Remove ald the files with the selected suffixes.
+	# Remove all the files with the selected suffixes.
 	for ext in "${exts[@]}"
 	do
 		for file in `ls *.$ext 2> /dev/null`
 		do
-			rm -rf *.aux *.bbl *.blg *.log *.out *.toc *.lot *.lof raven_user_manual.pdf
+			rm -rf *.aux *.bbl *.blg *.log *.out *.toc *.lot *.lof *.gz raven_user_guide.pdf
 		done
 	done
 }
 
+# load raven libraries
+source ../../scripts/establish_conda_env.sh --load --quiet
+
```

Comment:
why do we need to load the library here?

Reply:
If the user wants to build the user guide by itself, they can run `doc\user_gude> ./make_win.sh`. In the wiki it says this:

> If only a specific document needs to be built, navigate to the containing folder and type:
> 
> In GitBash:
> 
>   `./make_win.sh`
> In PowerShell:
> 
>   `bash.exe ./make_win.sh`
> For example:
> 
> `C:\projects> cd raven\doc\user_manual`
> `C:\projects\raven\user_manual> ./make_win.sh` (In GitBash) or `bash.exe make_win.sh` (In PowerShell)
> 

It is not needed if `doc> ./make_docs.sh` is run.

Reply:
In this case, we probably need to load the library in each make_win.sh.

Reply:
All of the other make_win.sh files run without loading the library. The user guide seems to be the only one that needs it. I'm guessing this is because the user guide takes TeX from RAVEN itself and the other manuals don't do this?